src/epub/text/chapter-26.xhtml

```diff
@@ -12,9 +12,7 @@
 				<h3 epub:type="title">Kilsip Has a Theory of His Own</h3>
 			</hgroup>
 			<p><abbr>Mr.</abbr> Calton sat in his office reading a letter he had just received from Fitzgerald, and judging from the complacent smile upon his face it seemed to give him the greatest satisfaction.</p>
-			<blockquote epub:type="z3998:letter">
-				<p>“I know,” wrote Brian, “that now you have taken up the affair, you will not stop until you find out everything, so, as I want the matter to rest as at present, I will anticipate you, and reveal all. You were right in your conjecture that I knew something likely to lead to the detection of Whyte’s murderer; but when I tell you my reasons for keeping such a thing secret, I am sure you will not blame me. Mind you, I do not say that I know who committed the murder; but I have suspicions⁠—very strong suspicions⁠—and I wish to God Rosanna Moore had died before she told me what she did. However, I will tell you all, and leave you to judge as to whether I was justified in concealing what I was told. I will call at your office some time next week, and then you will learn everything that Rosanna Moore told me; but once that you are possessed of the knowledge you will pity me.”</p>
-			</blockquote>
+			<p>“I know,” wrote Brian, “that now you have taken up the affair, you will not stop until you find out everything, so, as I want the matter to rest as at present, I will anticipate you, and reveal all. You were right in your conjecture that I knew something likely to lead to the detection of Whyte’s murderer; but when I tell you my reasons for keeping such a thing secret, I am sure you will not blame me. Mind you, I do not say that I know who committed the murder; but I have suspicions⁠—very strong suspicions⁠—and I wish to God Rosanna Moore had died before she told me what she did. However, I will tell you all, and leave you to judge as to whether I was justified in concealing what I was told. I will call at your office some time next week, and then you will learn everything that Rosanna Moore told me; but once that you are possessed of the knowledge you will pity me.”</p>
```

Comment:
In the page scans this is actually part of the preceding paragraph. Can you join them?

Reply:
Done in dbec3a53ea64fcf13c87fe088666e12d5373d2ea.